server/src/main/java/org/apache/iotdb/db/metadata/path/AlignedPath.java

```diff
@@ -201,6 +218,7 @@ public PartialPath copy() {
     result.fullPath = fullPath;
     result.device = device;
     result.measurementList = new ArrayList<>(measurementList);
```

Comment:
Add ` result.isUnderAlignedEntity = isUnderAlignedEntity;`

Reply:
MeasurementPath's copy() method need to be modified too.

Reply:
`isUnderAlignedEntity` is a filed of `MeasurementPath`, not `AlignedPath`.

Reply:
> MeasurementPath's copy() method need to be modified too.

done.